src/main/java/it/pn/frontend/e2e/pages/mittente/ApiKeyPAPage.java

```diff
@@ -414,8 +423,8 @@ public void siVisualizzaMessaggioErroreApiName() {
             getWebDriverWait(60).until(ExpectedConditions.visibilityOfElementLocated(nameErrorMessageANBy));
```

Comment:
inserire messaggio

Reply:
gia gestito con try and catch c'è il messaggio sotto